packages/emoji-mart-website/index.html

```diff
@@ -64,6 +64,7 @@
           ><select id="locale" class="flex-grow">
             <option value="en">English</option>
             <option value="ar">العربية</option>
+            <option value="cs">Czech</option>
```

Comment:
In this list we display the language in its native form. According to my search, it would be **čeština** for Czech, is that accurate?

Reply:
I'm sorry I overlooked that, you are right, **Čeština** is the correct word :) I'll send fix